rosidl_generator_py/package.xml

```diff
@@ -10,13 +10,28 @@
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
   <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
+  <buildtool_export_depend>rosidl_generator_c</buildtool_export_depend>
 
+  <build_depend>rmw</build_depend>
+  <build_depend>rmw_implementation</build_depend>
+  <build_depend>rosidl_generator_c</build_depend>
```

Comment:
When reading a manifest I find it most reasonable to first see all build dependencies, then all run dependencies and at the end the test dependencies. Within build and run its probably good to list the tool dependencies first.